mpisppy/cylinders/lagrangian_bounder.py

```diff
@@ -49,7 +49,8 @@ def lagrangian(self):
         if total == serial_number_sum:
             return bound
         elif self.cylinder_rank == 0:
-            print("WARNING: Lagrangian spokes out of snyc, consider changing mpisppy.cylinders.SPOKE_SLEEP_TIME")
+            print("WARNING: Lagrangian spokes out of snyc, consider changing the spoke_sleep_time option; "
+                  "see the documentation for more information")
```

Comment:
We should print the current value here, so the user doesn't have to guess at what it is.

Reply:
good point